addons/website/controllers/main.py

```diff
@@ -354,9 +354,9 @@ def theme_customize(self, enable=None, disable=None, get_bundle=False):
         if get_bundle:
             context = dict(request.context)
             return {
-                'web.assets_common': request.env["ir.qweb"]._get_asset('web.assets_common', options=context),
-                'web.assets_frontend': request.env["ir.qweb"]._get_asset('web.assets_frontend', options=context),
-                'website.assets_editor': request.env["ir.qweb"]._get_asset('website.assets_editor', options=context),
+                'web.assets_common': request.env["ir.qweb"]._get_asset_nodes('web.assets_common', options=context),
```

Comment:
@qsm-odoo This is not correct, it really wait some HTML code, thus the to_html method logic.
So maybe methods should be undeprecated rather than removed, dunno..